WcaOnRails/app/models/competition.rb

```diff
@@ -863,7 +863,18 @@ def days_until
 
   def has_date_errors?
     valid?
```

Comment:
Any idea what this line of code is doing? Is this just to trigger some `before_validation` callbacks?

Reply:
not entirely sure, but this is what triggers the competition validation error on submission (note you can still create the competition just fine with a date < 28 days):
![image](https://user-images.githubusercontent.com/11577241/62091262-144fc780-b23e-11e9-8fce-304b9547c46f.png)
The last validation error seen here.